oasislmf/execution/bash.py

```diff
@@ -1475,7 +1500,12 @@ def bash_params(
     else:
         bash_params['_get_getmodel_cmd'] = _get_getmodel_cmd
 
-    # Set fifo dirs
+    # Set custom gulcalc log statment checks, 
+        bash_params['custom_gulcalc_log_start'] = custom_gulcalc_log_start or analysis_settings.get('model_custom_gulcalc_log_start')
+        bash_params['custom_gulcalc_log_finish'] = custom_gulcalc_log_finish or analysis_settings.get('model_custom_gulcalc_log_finish')
```

Comment:
Indentation of comment.
```suggestion
        # Set custom gulcalc log statment checks, 
        bash_params['custom_gulcalc_log_start'] = custom_gulcalc_log_start or analysis_settings.get('model_custom_gulcalc_log_start')
        bash_params['custom_gulcalc_log_finish'] = custom_gulcalc_log_finish or analysis_settings.get('model_custom_gulcalc_log_finish')
```